src/Components/Admin/Admins/Form/index.jsx

```diff
@@ -97,7 +97,7 @@ const AdminsForm = ({ history }) => {
   return (
     <div>
       <form className={styles.form} onSubmit={handleSubmit(onSubmit, onInvalid)}>
-        <div className={styles.formContainer}>
+        <div className={styles.formContainer} testId="admin-form">
           <h1>{text}</h1>
           <fieldset className={styles.fieldset}>
             <Input
```

Comment:
In the lines 187, 188 and 192 i think you need to add the test id for the buttons like <Button type="confirm" testId="confirm-button"></Button>... i do this, but wait to confirm if it work's in my pr to change it